openshift-hack/e2e/annotate/rules.go

```diff
@@ -387,10 +376,6 @@ var (
 			`\[Feature:StorageProvider\]`,
 		},
 
-		// tests that don't pass under openshift-sdn multitenant mode
-		"[Skipped:Network/OpenShiftSDN/Multitenant]": {
-			`\[Feature:NetworkPolicy\]`, // not compatible with multitenant mode
-		},
 		// tests that don't pass under OVN Kubernetes
 		"[Skipped:Network/OVNKubernetes]": {
 			// ovn-kubernetes does not support named ports
```

Comment:
should the following skip be also removed?

https://github.com/openshift/kubernetes/blob/81c1851c0291160c33b40bc22f031293ef756879/openshift-hack/e2e/annotate/rules.go#L401-L403

Reply:
AFAIK `ibmroks` is still using Calico so they get their own separate set of skips.

(Well, chasing that bugzilla reference leads to OCPBUGS-8771 which claims that the bug is fixed in Calico now, so maybe this skip can be removed, but it's not related to openshift-sdn anyway.)

Reply:
I see, I will leave it up to you if you want to pursue that as a followup